Contributing/How-to-Contribute-Code.md

```diff
@@ -1,27 +1,42 @@
-    * Sign your message with `Signed-off-by: Random J Developer <random@developer.example.org>` to agree to [Developer Certificate of Origin (DCO)](https://developercertificate.org/) (to certify that you wrote or otherwise have the right to submit your contribution to the project.)
-    * Ensure your Pull-Request passes all checks and has no conflicts
-    * Click `Create Pull Request`
-1. Get credit for your work
-    * [Claim credit](https://github.com/moja-global/About-moja-global/blob/master/Contributing/How-to-Get-Credit-for-Your-Contribution.md) if you are not listed yet. 
+## Code Contributions
+
+We would like to thank you for your interest in contributing. You can find below how you can get started with knowing and contributing to moja global. The first few steps are always crucial but signify how you wish to involve with the community.
+
+1.  Get to know moja global:
+
+    -   Read the [Contributing guidelines](https://github.com/moja-global/About-moja-global/blob/master/CONTRIBUTING.md)
+    -   Read the [Code of Conduct](https://github.com/moja-global/.github/blob/master/CODE_OF_CONDUCT.md)
+    -   Review the [Coding Guidelines](https://github.com/moja-global/.github/blob/master/Governance/Coding-Guidelines.md) and the [Contribution Criteria](https://github.com/moja-global/.github/blob/master/Governance/Contribution-Criteria.md).
+    -   Review the bug-list, open issues, Project Board (if you want to code a new feature) or the [Strategic Plan](https://github.com/moja-global/About-moja-global/blob/master/Governance/Strategic-Plan.md). To start a new Project follow these [guidelines](https://github.com/moja-global/About-moja-global/blob/master/Contributing/How-to-Start-a-New-Project.md).) 
+    -   Contact the Coach, if available in the README document, or the Repository Maintainer by submitting an [Issue](https://github.com/moja-global/About-moja-global/blob/master/Contributing/How-to-Provide-User-Feedback.md) with an @mention.  
+
+2.  [Join the moja global organisation](https://github.com/moja-global/About-moja-global/blob/master/Contributing/How-to-Join-moja-global.md)  
+
+3.  [Fork the repository](https://help.github.com/en/articles/fork-a-repo) if you do not want to join moja global or want to start working before your request to join has been approved.
+
+4.  Follow [Vincent Driessen's Git Branching](https://nvie.com/posts/a-successful-git-branching-model/) System
+
+    -   Name your Branch with the same name using the Feature or Branch name with a [version number](https://github.com/moja-global/About-moja-global/blob/master/Contributing/How-to-Assign-a-Version.md).
+
+5.  Make your contribution
+
+    -   Make changes or additions to the code.
+    -  Ensure that your code meets the [coding style guidelines](https://github.com/moja-global/About-moja-global/blob/master/Governance/Coding-Guidelines.md).
+    - Perform relevant tests for the repository that you will be submitting the pull-request for.
```

Comment:
@HarshCasper please change the sentence to something which can clearly speaks how much testing for submitted code is important and required.